CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -18,7 +19,7 @@ import Vision
     }
     
     
-    @objc public weak var testingImageDataSource: TestingImageDataSource?
+    public var testingImageDataSource: TestingImageDataSource?
```

Comment:
I think we need to leave the `weak` in here to avoid a memory leak, but I'm not 100% sure

Reply:
it wont let me put the `weak` in there :-/ . i get the errors ```'weak' must not be applied to non-class-bound 'TestingImageDataSource'```.

Reply:
we should understand this more clearly. the "safe" thing to do here is to add the @objc and weak back and be done with it, but if you want to make a change it's going to require some deeper analysis

Reply:
im going to see if theres a workaround in the next commit to just add the @objc back. it's all because of that optional return